acme/acme.go

```diff
@@ -395,7 +387,7 @@ func dnsOverrideDelay(delay flaeg.Duration) error {
 
 func (a *ACME) buildACMEClient(account *Account) (*acme.Client, error) {
 	log.Debug("Building ACME client...")
-	caServer := "https://acme-v01.api.letsencrypt.org/directory"
+	caServer := "https://acme-v02.api.letsencrypt.org/directory"
```

Comment:
Is this fully backwards compatible? Is it something that we should be defaulting back to v1 somehow?

Reply:
@dtomcej It's not backward compatible.

That's why `getAccount` methods have been modified.
They can detect V1 Account format and delete is from the ACME structure.

Thus, Træfik reads the new configuration and create a new V2 ACME Account.

However, of course, ACME certificates generated in V1 are compatibles

Reply:
👍